util/find-doc-nits

```diff
@@ -254,12 +257,14 @@ my $option_re =
             [[:alpha:]](?:[-_[:alnum:]]*?[[:alnum:]])?
        )/x;
 
+# Normal symbol form
+my $symbol_re = qr/[[:alpha:]_][_[:alnum:]]*?/;
+
```

Comment:
I kept this near the function that uses it, why move it up?

Reply:
i was just trying to group variables together.  i guess that's not perl style.  I will put it back.